depthai_ros_msgs/include/depthai_ros_msgs/plugin/DaiCommonCameraBoardSocket.h

```diff
@@ -0,0 +1,4 @@
+enum class DaiCommonCameraBoardSocket : int32_t { AUTO = -1, RGB, LEFT, RIGHT };
```

Comment:
Any reason to create an empty message file instead of just a simple header with a class?

Reply:
This specific file was just wip.

For some of the other enum classes such as https://github.com/rapyuta-robotics/depthai_ros/pull/25/files#diff-ff4e859c1364e729519e8640509df9f4a6e01b1f0bf4a255da809a1cbeece3a7, empty msg file was added for renaming int32 to the message name.

Reply:
I see. Shouldn't the RawImgFrameType message be int32 in that case? Doesn't having it empty result in ROS message not having the type?

Reply:
I'm wondering what is best to handle just this empty enum. Maybe I will change these empty enum message types to native types.